content/2019.04.getting-started-with-elm.rst

```diff
@@ -0,0 +1,784 @@
+    view model =
+        div []
+            [ text "Hello "
+            , a [ href "#", onClick Switch ] [ text model.name ]
+            ]
+
+
+Enabling auto updates with ``elm-live``
+=======================================
+
+``elm reactor`` is good to get started but if you want hot-reloading of
+your app, you might want to setup ``elm-live``.
+
+Once installed, run:
+
+.. code-block:: bash
+
+    $ elm-live src/Main.elm
+
+If you have to use the debugger, you can use the ``--debug`` option:
+
+.. code-block:: bash
+
+    $ elm-live src/Main.elm -- --debug
+
+It will automatically generate an ``index.html`` file with the
+compiled JavaScript, and open it in your default Web browser.
+
+You can use the ``--output`` option to save the JavaScript in its own
+file and load it in the HTML yourself.
+
+First update the ``index.html`` to make it looks like:
+
+.. code-block:: html
+
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta charset="utf-8">
+        <title>Hello world</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="elm.js"></script>
+      </head>
+
+      <body>
+        <div id="sandbox"></div>
+        <script>
+            var app = Elm.Main.init({node: document.getElementById("sandbox")});
+        </script>
+      </body>
+    </html>
+
+
+Then you can run elm-live with the ``--output`` option:
+
+.. code-block:: bash
+
+    $ elm-live src/Main.elm -- --debug --output elm.js
+
+Now each time you will update your Elm code it will refresh the app in
+the browser.
+
+
+Handling a second event
+=======================
+
+Let's add an input to let people choose who to great.
+
+.. code-block:: elm
+
+    module Main exposing (main)
+
+    import Browser
+    import Html exposing (..)
+    import Html.Attributes exposing (..)
+    import Html.Events exposing (..)
+
+
+    type Msg
+        = Switch
+        | NewName String
+
+
+    type alias Model =
+        { name : String }
+
+
+    main =
+        Browser.sandbox { init = { name = "Rémy" }, view = view, update = update }
+
+
+    update : Msg -> Model -> Model
+    update msg model =
+        case msg of
+            Switch ->
+                if model.name == "Rémy" then
+                    { model | name = "Séverine" }
+                else
+                    { model | name = "Rémy" }
+
+            NewName new_name ->
+                { model | name = new_name }
+
+
+    view : Model -> Html Msg
+    view model =
+        div []
+            [ text "Hello "
+            , a [ href "#", onClick Switch ] [ text model.name ]
+            , br [] []
+            , input
+                [ onInput NewName
+                , value model.name
+                ]
+                []
+            ]
+
+The ``NewName`` event will be emitted with the content of the input each time we type in it.
+
+
+Conclusion
+==========
+
+That's about it. Now that you understood how the events update mecanism works and how
```

Comment:
I would probably say `Now that you understand`, and `event update` instead of `events update`. Also `mechanism`.